### PR TITLE
DL-3007 SBT Update

### PR DESCRIPTION
**Pull request by Sir-Iyke**
DL-3007 SBT Update

**New feature**

Upgraded SBT and Scala dependencies

## Checklist

*Reviewee* (Iyke)
 - [X]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [X]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [X]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [X]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [X]  I've run a dependency check to ensure all dependencies are up to date

*Reviewer* (Paul Anderson)
 - [x]  I've confirmed that every effort has been made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've confirmed appropriate tests has been included with any code added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've confirmed code was added using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    unmanagedSourceDirectories in IntegrationTest :=  (baseDirectory in IntegrationTest)(base => Seq(base / "it")).value,
+    parallelExecution in IntegrationTest := false
+  )
+  .settings(inConfig(TemplateTest)(Defaults.testSettings): _*)
+  .settings(inConfig(TemplateItTest)(Defaults.itSettings): _*)
+  .configs(IntegrationTest)
+  .settings(
+    resolvers += Resolver.bintrayRepo("hmrc", "releases"),
+    resolvers += Resolver.jcenterRepo
+  )
```
Comment (paulanderson1234) [inline build.sbt:58]: Need to add entry for .disablePlugins(JUnitXmlReportPlugin) to sbt file otherwise the build will fail

**Comment by paulanderson1234** [comment]
Also need additional config changes to service-manager-config and build-jobs to support this PR

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -16,9 +9,9 @@ private object AppDependencies {
     ws,
     "uk.gov.hmrc" %% "bootstrap-play-26" % "1.7.0",
     "uk.gov.hmrc" %% "govuk-template" % "5.54.0-play-26",
-    "uk.gov.hmrc" %% "play-ui" % "8.8.0-play-26",
+    "uk.gov.hmrc" %% "play-ui" % "8.9.0-play-26",
     "uk.gov.hmrc" %% "play-partials" % "6.10.0-play-26",
```
Comment (paulanderson1234) [inline project/AppDependencies.scala:13]: This can probably go up to 6.11.0-play-26

Comment (paulanderson1234) [inline project/AppDependencies.scala:13]: play-partials can proably go up a version to 6.11.0-play-26

File: build.sbt (hunk 1 above)
Comment (Sir-Iyke) [reply build.sbt:58]: Done

**Comment by Sir-Iyke** [comment]
https://github.com/hmrc/service-manager-config/pull/2143

**Pull request merged**